src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -12325,7 +12325,9 @@ unsafe class C { }
             verifier.VerifyDiagnostics();
         }
 
-        [Theory, CombinatorialData, WorkItem(65594, "https://github.com/dotnet/roslyn/issues/65594")]
+        [ConditionalTheory(typeof(NotOnAnyMono), Reason = "https://github.com/dotnet/runtime/issues/118568")]
```

Comment:
Btw, your PR description says "closes: https://github.com/dotnet/runtime/issues/118568" which would close the runtime issue when this PR is merged. Is that intended? That way it might not get triaged.